guake/notebook.py

```diff
@@ -352,8 +354,16 @@ def new_page(self, directory=None, position=None):
         # this is needed to initially set the last_terminal_focused,
         # one could also call terminal.get_parent().on_terminal_focus()
         self.terminal_attached(terminal)
+        self.hide_if_one_tab()
         return root_terminal_box, page_num, terminal
 
+    def hide_if_one_tab(self):
+        if self.guake.settings.general.get_boolean("window-tabbar"):
```

Comment:
Add docstring to describe what this done.

Reply:
there's a lot of functions in the same file missing docstrings adjacent to this one but I'll add this one.

Reply:
Yeah, but we could polish our future commit code ！